webpack/ForemanInventoryUpload/Components/PageHeader/components/SettingsWarning/SettingsWarning.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const SettingsWarning = ({
+  autoUpload,
+  hostObfuscation,
+  isCloudConnector,
+}) => {
+  if (!isCloudConnector) {
+    return null;
+  }
+
+  const alerts = [];
+  if (!autoUpload) {
+    alerts.push(
+      <Alert
+        key="auto-upload"
+        variant="warning"
+        title={__(
+          "Cloud Connector has been configured however the inventory auto-upload is disabled, it's recommended to enable it"
+        )}
+      />
+    );
+  }
+  if (hostObfuscation) {
+    alerts.push(
+      <Alert
+        key="obfuscating-host"
+        variant="warning"
+        title={__(
+          "Cloud Connector has been configured however obfuscating host names setting is enabled, it's recommended to disable it"
+        )}
+      />
+    );
+  }
+  if (!autoUpload || hostObfuscation) {
+    return <div className="settings-alert">{alerts}</div>;
+  }
+  return null;
```

Comment:
can you add the opposite condition to line 12 in order to `return null` early?
```suggestion
    return <div className="settings-alert">{alerts}</div>;
```